### PR TITLE
Removes the stroke setting the opacity to 0. Adds specs

### PR DESCRIPTION
**Pull request by javierarce**
This PR removes the stroke of the points that don't pass a certain condition:

```ROW_COUNT / (CARD_WIDTH * CARD_HEIGHT) > STROKE_PX_LIMIT```

where 

* `CARD_WIDTH` = 288
* `CARD_HEIGHT` = 170
* `CARD_WIDTH` * `CARD_HEIGHT` = 48,960
* `STROKE_PX_LIMIT` = 0.04

Therefore if a dataset contains more than ≈ **2,000** rows, the strokes will be hidden.

The method used to disable the strokes consists in setting the opacity of the strokes to zero in the default CSS.

Updated later: instead of hiding the stroke, we'll modify the width and size of the markers. See comment thread for an updated example.

screen shot 2015-07-23 at 11 33 27

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5232/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5275/))

**Comment by javierarce** [comment]
retest this please

**Comment by javierarce** [comment]
cc: @javisantana, @saleiva, @andrewxhill

**Comment by javierarce** [comment]
@xavijam, can you review the PR, please?

**Comment by saleiva** [comment]
I don't think just removing the stroke is a good idea. We should change the size of the points, and not removing but making the stroke thinner

**Comment by javierarce** [comment]
LOL. You didn't say anything in the original ticket.

**Comment by saleiva** [comment]
:(. Well the logic is the same, right? Also, what happens when you apply? Do the wizard have strokes?

**Comment by javierarce** [comment]
Haha, no problem. This would only affect the preview. And yes, the logic will be quite similar, so it shouldn't add extra work.

Regarding the stroke width, we have this issue here: 
https://github.com/CartoDB/cartodb/issues/4468… so I'm not sure what to do. We could apply @javisantana's calculation to change the preview… or we could do what Andrew suggests.

**Comment by javierarce** [comment]
Ok, after talking with Sergio we're going to try a different approach: instead of hiding the stroke, we'll reduce the size of the markers (if the dataset contains points) and make the stroke thinner. I'll update the description of the PR once I've updated the branch with the fixes.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5301/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5306/))

**Comment by javierarce** [comment]
screen shot 2015-07-23 at 15 36 06

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5310/))

**Comment by saleiva** [comment]
AWESOME

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ module.exports = cdb.core.View.extend({
   },
 
   _getGeometryType: function() {
-    var geometry_types = this.table.data().table.get("stats_geometry_types");
+    var geometry_types = this.table.data().table.get("geometry_types");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js]: camelCase please :), look at line 34 ^^

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js [hunk 2]
```diff
@@ -208,6 +219,20 @@ module.exports = BaseDialog.extend({
...
+    return template;
+  },
+
+  _generateLayerDefinition: function(column) {
+
+    var type = column.get("visualizationType");
+    var sql = column.get("sql");
+    var css = this._setupCSS(column.get("css"), column.get("geometryType"));
+
+    var api_key  = this.user.get("api_key");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js]: we should use camel case for our internal variables.

**Comment by xavijam** [comment]
Cheap things commented, :es:.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5315/))

**Comment by javierarce** [comment]
Thanks guys, I'll deploy to production as soon as the build goes green.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5318/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5322/))

**Pull request merged**